src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double firstSide;
+    private final double secondSide;
+    private final double height;
+
+    public IsoscelesTrapezoid(double longerBase, double shorterBase, double height, String color) {
+        super(color);
+        this.firstSide = longerBase;
+        this.secondSide = shorterBase;
+        this.height = height;
+    }
+
+    @Override
+    public double calculateArea() {
+        return ((secondSide + firstSide) * height) / 2;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: isosceles trapezoid, area: " + calculateArea() + " sq"
```

Comment:
In the draw() method, 'color' is directly accessed which may not exist. Consider using getColor() method if it exists in the parent Figure class.

Reply:
done